manifests/repo.pp

```diff
@@ -33,17 +33,21 @@
 
   case $::osfamily {
     'Debian': {
-      if !defined(Class['apt']) {
-        class { 'apt': }
-      }
+      include ::apt
+
+      Class['apt::update'] -> Package<| |>
```

Comment:
This is dangerous and shouldn't be done. As far as i know the best practice way is:

```puppet
  Apt::Source['logstash'] -> Package<|tag == 'logstash'|>
  Class['Apt::Update'] -> Package<|tag == 'logstash'|>
```
Then add the following to the package resources which might depend on the repository:

```puppet
tag => 'logstash',
```